addons/mass_mailing/models/mailing.py

```diff
@@ -200,7 +200,7 @@ def _compute_total(self):
         for mass_mailing in self:
             total = self.env[mass_mailing.mailing_model_real].search_count(mass_mailing._parse_mailing_domain())
             if mass_mailing.ab_testing_pc < 100:
-                total = int(total / 100.0 * mass_mailing.ab_testing_pc)
+                total = int(total / 100.0 * mass_mailing.ab_testing_pc) or 1 if total else 0
```

Comment:
Could we write it like this:

```py
if total and mass_mailing.ab_testing_pc < 100:
    total = max(int(total / 100.0 * mass_mailing.ab_testing_pc), 1)
```

 ?
(Same for the other one below).

Makes it a bit easier to see that we want at least 1 for the total when the total is not 0 in the first place.